### PR TITLE
mysql_example: Mysql to Drupal 8 migration example module added.

### PR DESCRIPTION
**Pull request by pranitjha**

### CODE REVIEW

File: modules/examples/mysql_migrate/src/Plugin/migrate/source/Comments.php [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      'created' => $this->t('Created'),
+    ];
+
+    return $fields;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function prepareRow(Row $row) {
```
Comment (joshirohit100) [inline modules/examples/mysql_migrate/src/Plugin/migrate/source/Comments.php]: Do we need this?

File: modules/examples/mysql_migrate/src/Plugin/migrate/source/Content.php [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    ];
+
+    return $fields;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function prepareRow(Row $row) {
+    // Perform extra pre-processing for keywords terms, if needed.
```
Comment (joshirohit100) [inline modules/examples/mysql_migrate/src/Plugin/migrate/source/Content.php]: Same here

File: modules/examples/mysql_migrate/src/Plugin/migrate/source/Taxonomy.php [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    ];
+
+    return $fields;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function prepareRow(Row $row) {
+    // Perform extra pre-processing for keywords terms, if needed.
```
Comment (joshirohit100) [inline modules/examples/mysql_migrate/src/Plugin/migrate/source/Taxonomy.php]: Same here

File: modules/examples/mysql_migrate/src/Plugin/migrate/source/Users.php [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+namespace Drupal\mysql_migrate\Plugin\migrate\source;
+
+use Drupal\migrate\Plugin\migrate\source\SqlBase;
+use Drupal\migrate\Row;
+
+/**
+ * Source plugin for the users.
+ *
+ * @MigrateSource(
+ *   id = "mysql_users",
```
Comment (joshirohit100) [inline modules/examples/mysql_migrate/src/Plugin/migrate/source/Users.php]: Plugin name not looks right  (or as other)

File: modules/examples/mysql_migrate/src/Plugin/migrate/source/Users.php [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+    ];
+
+    return $fields;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function prepareRow(Row $row) {
+    // Perform extra pre-processing for keywords terms, if needed.
```
Comment (joshirohit100) [inline modules/examples/mysql_migrate/src/Plugin/migrate/source/Users.php]: Same here

**Comment by joshirohit100** [comment]
Looks good to me. One point - can we just change the source plugin id names to something else instead more generic? Like - **mg_usr_sql_example**. cc @amitgoyal for review

**Comment by amitgoyal** [comment]
@pranitjha Let's make these minor changes and have plugin names like mg_sample_sql_ENTITY.

**Comment by joshirohit100** [comment]
looks fine to me.

**Pull request merged**